.github/workflows/build_tests.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
       run: |
         python3 xpk.py cluster create-pathways --cluster $TPU_CLUSTER_NAME --tpu-type=v4-8  --num-slices=2 \
         --zone=us-central2-b --default-pool-cpu-machine-type=n1-standard-16 --default-pool-cpu-num-nodes=16 \
-        --reservation='${{ secrets.GCP_TPU_V4_RESERVATION }}' --enable-workload-identity --enable-gcsfuse-csi-driver
+        --reservation='${{ secrets.GCP_TPU_V4_RESERVATION }}' --enable-workload-identity --enable-gcsfuse-csi-driver --custom-cluster-arguments="${CLUSTER_ARGUMENTS}"
```

Comment:
Do we not create separate tests for creating cluster with filestore storage or fuse storage? Or do we leave it until later - when we will use them in workloads?

Reply:
Running tests with filestore is time consuming. I will leave it as it is